### ISSUE TITLE
Changing a form handle doesn't result in the submissions table being renamed, breaking submissions

### ISSUE THREAD

**Issue by tomdavies** [solspace/craft-freeform#376]
**Describe the bug or issue you're experiencing**

Changing a form handle doesn't result in the submissions table being renamed, but FF builds the name of the table to insert submissions into from the form handle, resulting in a DB exception when attempting to submit the renamed form

**Steps to reproduce**

1. Create a form & save it
2. Change the form handle & save again
3. Attempt to make a form submission
4. A DB exception is thrown:

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db.craft_freeform_submissions_oldformhandle_1' doesn't existSQLSTATE[42S02]: Base table or view not found: 1146 Table 'db.craft_freeform_submissions_oldformhandle_1' doesn't exist
```

<img width="1680" alt="Screenshot 2022-06-23 at 10 31 01" src="https://user-images.githubusercontent.com/136308/175267835-9a475ab8-baff-40b6-be69-af3c5df1df15.png">

**Expected behavior**

Changing the form handle should not break form submissions

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.0.5.1
- Freeform Version: v4.0.0-beta.12
- Freeform Edition: Pro
- Fresh Install or Upgrade: upgrade from v4.0.0-beta.11

**Comment by kjmartens** [comment]
@tomdavies Sorry for the trouble you're experiencing. I wonder if you're experiencing the issue from before the fix we made in 4.0.0-beta.9. Are you able to continually reproduce this error if you create a brand new form right now and rename the handle, etc? I tried this again now and no matter what I did, it seemed to always behave correctly for me. 🙂

**Comment by tomdavies** [comment]
@kjmartens quite possibly, as this was with an older form created on an early beta. The first thing I tried when I hit the big was updating to the latest beta and I seemed to still be getting the issue. Creating a new form doesn't display the issue ¯\_(ツ)_/¯

TBH if it's fixed for new forms (which it seems to be) I'm happy to close as already fixed
